packages/date-picker/src/panel/date.vue

```diff
@@ -52,15 +52,19 @@
             <button
               type="button"
               @click="prevYear"
+              @mousedown="prevYearMousedown"
```

Comment:
In order to prevent the buttons from becoming focused when they're disabled, subscribe to `mousedown` and cancel the event in that case